raindrops_updated.rb

```diff
@@ -0,0 +1,41 @@
+class Raindrops
+  def self.convert(num)
+    factors = ""
+    count = num
+    (1..count).each do |i|
+      if(num % i == 0)
+        num = num / i
+        factors += i.to_s + " "
+      end
+    end
+    final_output = Raindrops.result(factors)
+    if final_output == ""
+      return count.to_s
+    else
+      return final_output
+    end
+  end
+
+  def self.result(factors_called)
```

Comment:
If this method is for internal use only inside the class only then why it is class method? Why we can't use it as private method?